src/pages/ViewerPage.vue

```diff
@@ -1,18 +1,145 @@
 <template>
   <v-content>
-    Simulation viewer
+    <v-navigation-drawer
+      app
+      class="view-controls"
+      clipped
+      dark
+      permanent
+    >
+      <v-card
+        color="transparent"
+        flat
+        tile
+      >
+        <v-card-subtitle
+          class="align-center d-flex grey--text py-2"
+        >
+          Viewer Controls
+          <v-spacer />
+          <div class="expansion-controls">
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-btn
+                  v-if="panel.length < 3"
+                  icon
+                  small
+                  @click="all"
+                  v-on="on"
+                >
+                  <v-icon size="16">
+                    mdi-arrow-expand-vertical
+                  </v-icon>
+                </v-btn>
+                <v-btn
+                  v-else
+                  icon
+                  disabled
+                  small
+                >
+                  <v-icon size="16">
+                    mdi-arrow-expand-vertical
+                  </v-icon>
+                </v-btn>
+              </template>
+              <span>Expand All</span>
+            </v-tooltip>
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-btn
+                  v-if="panel.length > 0"
+                  icon
+                  small
+                  @click="none"
+                  v-on="on"
+                >
+                  <v-icon size="16">
+                    mdi-arrow-collapse-vertical
+                  </v-icon>
+                </v-btn>
+                <v-btn
+                  v-else
+                  icon
+                  disabled
+                  small
+                >
+                  <v-icon size="16">
+                    mdi-arrow-collapse-vertical
+                  </v-icon>
+                </v-btn>
+              </template>
+              <span>Collapse All</span>
+            </v-tooltip>
+          </div>
+        </v-card-subtitle>
+        <v-divider />
+        <v-expansion-panels
+          v-model="panel"
+          accordion
+          flat
+          multiple
+          tile
+        >
+          <v-expansion-panel
+            v-for="(item,i) in viewPanels"
+            :key="i"
+            class="grey darken-3"
+          >
+            <v-expansion-panel-header>
+              {{ item.header }}
+            </v-expansion-panel-header>
+            <v-expansion-panel-content>
+              Controls here
+              <div class="d-flex panel-footer pt-3">
+                <v-spacer />
+                <v-btn
+                  color="primary"
+                  small
+                >
+                  Apply
+                </v-btn>
+              </div>
+            </v-expansion-panel-content>
```

Comment:
We'll probably want make components for the contents of these controls, but this is fine for now.

Reply:
I'm completely onboard with this approach. I wanted to see if there was the ability to put `<componentName />` in the array for controls, but you need to also register that component and a couple errors are thrown. Not sure the best way to handle this.